pyproject.toml

```diff
@@ -26,7 +26,8 @@ dependencies = [
 [project.optional-dependencies]
 test = [
     "pytest",
-    "sympy"
+    "sympy",
```

Comment:
Quippy-ASE is a bit too big a dependency to add. Could the committee models be tested with one of the simpler calculators already built in to matscipy? Alternatively the test could check if the import succeeds and skip if not.

Reply:
Thanks for the suggestion. Sure, we can get rid of the `quippy`-dependency by using simple calculator. With `ase` being already a dependency, I was thinking of using a committee of `LennardJones`-Calculators (with varying parameters) instead.